src/objects/entities/Ben3.ts

```diff
@@ -107,10 +107,19 @@ class Ben3 extends Entity {
 
   jump() {
     if (this.sensorData.bottom.size >= 1) {
-      // @ts-expect-error todo
-      this.gameObject.applyForce({ x: 0, y: -0.3 });
-      // @ts-expect-error todo
-      this.head.applyForce({ x: 0, y: -0.3 });
+      const { body: Body } = this.scene.matter;
+
+      const { centerX, centerY } = this.gameObject.getBounds();
+      const position = { x: centerX, y: centerY };
+      Body.applyForce(this.gameObject.body, position, {
+        x: 0,
+        y: -0.05 * this.gameObject.body.mass,
```

Comment:
Did we not want to multiply this by the mass of the head or does the body include head mass?

Reply:
force is applied to head on line 119

its the same force, which is 5% of the objects mass